src/toolbelt/date.clj

```diff
@@ -220,6 +308,64 @@
   (t/months n))
 
 
+(defn to-map
+  "Returns a map with keys and values representing the supplied `date`. The returned map includes
+  the following keys: #{:year :month :day :hour :minute :second :millisecond :day-of-week :week-of-year}
+  with corresponding number values."
+  [date]
+  (let [[year month day hour minute second millis dow woy] ((juxt t/year t/month t/day t/hour t/minute t/second t/milli t/day-of-week t/week-number-of-year)
+                                                             (c/to-date-time date))]
+    {:year         year
+     :month        month
+     :day          day
+     :hour         hour
+     :minute       minute
+     :second       second
+     :millisecond  millis
+     :day-of-week  dow
+     :week-of-year woy}))
+
+
+;; =============================================================================
+;; Coercions
+;; =============================================================================
+
+
+(defn from-map
+  "Returns a date given a map with keys and values representing a date time. Considers the following
+  keys: #{:year :month :day :hour :minute :second :millisecond} with corresponding number values
+  when creating the date."
+  [{:keys [year month day hour minute second millisecond]}]
+  (let [[y m d h min sec mill] (mapv (fnil identity 0) [year month day hour minute second millisecond])]
+    (to-date (t/date-time y m d h min sec mill))))
+
+
+(defn to-unix-time
+  "Return the number of seconds after the Unix time
+  Arity 1: returns the number of seconds after Unix time.
+  Arity 2: returns the time in `unit` after Unix time: #{:millis :seconds}"
+  ([dt]
+   (to-unix-time dt :millis))
```

Comment:
docstring above leads me to believe this should be `:seconds`, not `:millis`

Reply:
doc string is wrong, I'll update